drip/models.py

```diff
@@ -224,6 +251,21 @@ def set_booleans(self, field_value):
         return field_value
 
     def filter_kwargs(self, qs, now=datetime.now):
+        """
+        Returns a dictionary {field_name: field_value} where:
+        * field_name is self.annotated_field_name in addition to
+        self.lookup_type in the form FIELD_NAME__LOOKUP.
+        * field_value is the result of passing self.field_value through
+        parsing methods.
+
+        The resulting dict can be used to apply filters over querysets.
+
+        For example:
+            queryset.filter(**obj.filter_kwargs(datetime.now()))
+        :param qs: queryset  #TODO: remove this parameter since is not used.
```

Comment:
What about the comment `#TODO: remove this parameter since is not used.` here?

Reply:
I wonder why we are receiving the qs parameter since we are not using it inside the method.

Reply:
You're right, we need to take a look at this

Reply:
@jiss2891 Remove the documentation of this parameter since you're fixing it in another PR

Reply:
Sure, thanks!